pywren/wrenconfig.py

```diff
@@ -38,7 +38,9 @@ def load(config_filename):
                 config_filename, res['s3']['bucket']))
     if 'storage_backend' not in res:
         res = patch_storage_config(res)
-
+    elif res['storage_backend'] == 's3':
```

Comment:
could you remind me why we need this ? I remember @ericmjonas made the `patch_storage_config` for a specific reason. Also a comment here on what is going on would be useful ?

Reply:
This is from #162. When we added the storage API, we added some field to`pywren_config` but didn't include them in the default config that's created when someone installs pywren.